cli/cmd/dashboard.go

```diff
@@ -98,5 +132,5 @@ func init() {
 	// This is identical to what `kubectl proxy --help` reports, `--port 0`
 	// indicates a random port.
 	dashboardCmd.PersistentFlags().IntVarP(&dashboardProxyPort, "port", "p", 0, "The port on which to run the proxy. When set to 0, a random port will be used.")
-	dashboardCmd.PersistentFlags().BoolVar(&dashboardSkipBrowser, "url", false, "Display the Conduit dashboard URL in the CLI instead of opening it in the default browser")
+	dashboardCmd.PersistentFlags().StringVar(&dashboardShow, "show", "conduit", "Open a dashboard in a browser or show URLs in the CLI. Must be one of: conduit, grafana, url.")
```

Comment:
This is unfortunately a breaking change for the CLI. My preference would be to keep the `--url` flag, and use that to control whether or not we open the browser, regardless of which dashboard we're opening. And then maybe we could controller which dashboard is opened via a separate boolean `--grafana` flag? Does that seem feasible?

Reply:
It is feasible. I did consider this approach, but decided a single `--show` flag was more user-friendly, at the expense of breaking backwards compatibility (particularly because we're not providing any interface compatibility guarantees yet).

Reply:
Yeah, totally agree that we're not worried about providing backwards compatibility at this stage.

I guess another way to put it is that, in my mind, we're trying to control two different binary things via a single string flag, which feels more complicated than two separate boolean flags.

Fwiw, we originally settled on the `--url` flag based on how the `minikube service` command works:

```
$ minikube help service | grep '\-\-url'
      --url                Display the kubernetes service URL in the CLI instead of opening it in the default browser
```

So it's possible I'm just attached to having a `--url` flag based on my usage of minikube.